docs/conf.py

```diff
@@ -8,9 +8,12 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/"))
```

Comment:
ugh... isn't there a better way?

Maybe setting the `PYTHONPATH` env var?

Reply:
I'm not entirely sure @BeyondEvil. I made that change as per https://github.com/pytest-dev/pytest-html/blob/master/docs/conf.py#L7 and also some results on the internet. It seems like this is the preferred way to go about doing this

Reply:
Fair enough @gnikonorov

Reply:
Are you ok with me merging this in then @BeyondEvil?

Reply:
Well sure, on one condition. If you, once it's rebased and updated, approve the flaky windows PR. 😛 @gnikonorov